src/reference/00-Getting-Started/07A-Scopes.md

```diff
@@ -354,3 +353,42 @@ has three axes). The entire expression
 Simply `packageOptions` is also a key name, but a different one (for keys
 with no in, a scope is implicitly assumed: current project, global
 config, global task).
+
+#### Build-wide settings
+
+An advanced technique for factoring out common settings
+across subprojects is to define the settings scoped to `ThisBuild`.
+
+If a key is not defined scoped to a particular subproject,
+sbt will look for it in `ThisBuild` as a fallback.
+Using the mechanism, we can define a build-wide default setting for
+frequently used keys such as `version`, `scalaVersion`, and `organization`.
+
+For convenience, there is `inThisBuild(...)` function that will
+scope both the key and the body of the setting expression to `ThisBuild`.
+
+```scala
+lazy val root = (project in file("."))
+  .settings(
+    inThisBuild(List(
+      // Same as:
+      // organization in ThisBuild := "com.example"
+      organization := "com.example",
+      scalaVersion := "$example_scala_version$",
+      version      := "0.1.0-SNAPSHOT"
+    )),
+    name := "Hello",
+    publish := (),
+    publishLocal := ()
+  )
```

Comment:
This is confusing to me because `inThisBuild` is used *within* the `.settings` of a project. Is it possible to write this global definitions *outside* of a project definition?

Also, it would be super interesting to mention `inConfig` as well.